src/main/java/org/folio/circulation/domain/RequestAndRelatedRecords.java

```diff
@@ -124,6 +132,14 @@ String getDestinationItemId() {
     return moveRequestRecord != null ? moveRequestRecord.getDestinationItemId() : null;
   }
 
+  String getDestinationItemInstanceId() {
+    return moveRequestRecord != null ? moveRequestRecord.getDestinationItemInstanceId() : null;
```

Comment:
try to avoid returning nulls if possible

Reply:
I've deleted this method.